Dockerfile

```diff
@@ -1,2 +1,39 @@
-FROM derjudge/archlinux:latest
+FROM vcatechnology/base-archlinux:latest
```

Comment:
I wasn't able to do a successful pacman update from that image - some errors relating to certificate verify locations which I wasnt able to solve (the suggestions where to re-install arch using an updated iso). The last update was over a year ago, so I created an updated image using the same method as base/archlinux - the [mkimage-arch ](https://github.com/dotcloud/docker/blob/master/contrib/mkimage-arch.sh) script from docker.

I figure we can update the base image - if any pacman updates end up requiring manual intervention/fixups